src/renderer/common/redux/sagas/browse.ts

```diff
@@ -18,12 +18,12 @@ import { apiSaga } from "./api";
 const filename_ = "readium-desktop:renderer:redux:saga:opds-browse";
 const debug = debug_(filename_);
 
-type TA = apiActions.result.TAction<TReturnPromiseOrGeneratorType<TApiMethod["opds/browse"]>>;
+type TA = apiActions.result.TAction<TReturnPromiseOrGeneratorType<TApiMethod["browser/browse"]>>;
 
 export function* opdsBrowse(link: string, REQUEST_ID: string) {
 
     debug("opds-browse", link);
-    yield apiSaga("opds/browse", REQUEST_ID, link);
+    yield apiSaga("browser/browse", REQUEST_ID, link);
```

Comment:
Can you please explain the rationale for the change from `opds/browse` to `browse/browse`? If the intent is to signal that this API will / can work with any HTTP request/response cycle, then I understand, but `browse/browse` looks like a typo rather than an intentional entry point in the API surface. Would `http/browse` be preferable, or is that a misnomer too?

Reply:
...furthermore, there is a file renaming from `opdsBrowse.ts` to `browse.ts` which signals the generic nature of the API, but the function is still named `opdsBrowse()`:

```
export function* opdsBrowse(link: string, REQUEST_ID: string) {
...
}
```

Reply:
...I also note that the `IBrowserResultView` signals a generic HTTP request/response use-case, yet its object shape / model seems to be tightly-coupled with OPDS:

```
export interface IBrowserResultView {
    opds?: IOpdsResultView;
    problemDetails?: IOpdsProblemDetailsResultView;
}
```

Reply:
thanks 

`browser/browse` replace with `httpbrowser/browse` instead